src/pages/resources/howtos/running-network-infrastructure.md

```
@@ -0,0 +1,184 @@
+   // ...
+ });
+```
+
+!!! info Hardening your server against unintended use
+We've shown how to configure the server without authentication. In a production scenario, you'll likely want to authenticate incoming connections to the server because:
+
+* Unauthorized requests to the bootstrap endpoint could leak details about what devices are running what hApps, and
+* Unauthorized requests to the signal/relay endpoint allow users of other hApps to freeload on your server's bandwidth.
+
+We plan to discuss [authentication options](https://github.com/holochain/sbd/blob/main/spec-auth.md) in the future.
+!!!
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)